utils/src/main/scala/com/salesforce/op/utils/spark/OpSparkListener.scala

```diff
@@ -252,4 +296,38 @@ object StageMetrics {
       shuffleRecordsWritten = tm.shuffleWriteMetrics.recordsWritten
     )
   }
+  def zero(): StageMetrics = StageMetrics(0,
+    0,
```

Comment:
please ident it compactly. you can either use several lines or turn off the scalastyle on the line